packages/cozy-client/src/models/country/countries.js

```diff
@@ -433,3 +276,68 @@ export const checkCountryCode = code => {
   )
   return !!foundCountry
 }
+
+/**
+ * Get country name by its ISO 3166-1 (alpha-2, alpha-3 or numeric code)
+ *
+ * @param {string|number} code - ISO 3166-1 alpha-2, alpha-3 or numeric code
+ * @param {object} options - Options
+ * @param {string} options.lang - Language of country names
+ * @returns {string|number} - Country name or code if not found
+ */
+export const getCountryNameByCodeISO = (code, { lang } = { lang: 'en' }) => {
+  /**
+   * @param {object} params - Params
+   * @param {import('../../types').CountryISO} params.item - Country ISO object
+   * @param {string|number} params.code- ISO 3166-1 alpha-2, alpha-3 or numeric code
+   * @returns {boolean} - True if the code is valid, false otherwise
+   */
+  const findCountryISO = ({ item, code }) => {
+    const type = typeof code
+    if (type === 'string') {
+      // @ts-ignore - "code" is a string
+      const codeUpperCase = code.toUpperCase()
+      return item.code2 === codeUpperCase || item.code3 === codeUpperCase
+    } else if (type === 'number') {
+      return item.number === code
+    } else {
+      return false
+    }
+  }
```

Comment:
Is there a reason to have this function inside the getCountryNameByCodeISO ?

Reply:
Agreed, looks like a mistake

Reply:
Not really, as it's a `.find` function below I thought it would be a good idea to scope it to the parent function.
But during coffee we noticed a major problem with the condition on the "number" type, which will never be true...

I'll ping you once it's fixed

Reply:
Now that's fixed, if there's no alert I'll merge after the CI 👍